.github/workflows/deploy-build-from-pr.yml

```diff
@@ -165,7 +165,7 @@ jobs:
       - name: Tag and push to Jfrog Registry
         if: github.event.workflow.name == 'cwag-workflow' && steps.save_metadata.outputs.skipped == 'false'
         env:
-          DOCKER_REGISTRY: "${{ secrets.DOCKER_REGISTRY }}"
+          DOCKER_REGISTRY: "orc8r-ci.artifactory.magmacore.org"
           DOCKER_USERNAME: "${{ secrets.DOCKER_USERNAME }}"
```

Comment:
DOCKER_REGISTRY was pointing to the old fb artifactory... So you need to update the USERNAME AND PASSWORD too to:
`       DOCKER_USERNAME: "${{ secrets.JFROG_USERNAME }}"
          DOCKER_PASSWORD: "${{ secrets.JFROG_PASSWORD }}"`